client/consensus/manual-seal/Cargo.toml

```diff
@@ -22,20 +22,28 @@ parking_lot = "0.10.0"
 serde = { version = "1.0", features=["derive"] }
 assert_matches = "1.3.0"
 
-sc-client-api = { path = "../../../client/api", version = "2.0.0-rc6" }
-sc-transaction-pool = { path = "../../transaction-pool", version = "2.0.0-rc6" }
-sp-blockchain = { path = "../../../primitives/blockchain", version = "2.0.0-rc6" }
-sp-consensus = { package = "sp-consensus", path = "../../../primitives/consensus/common", version = "0.8.0-rc6" }
-sp-inherents = { path = "../../../primitives/inherents", version = "2.0.0-rc6" }
-sp-runtime = {  path = "../../../primitives/runtime", version = "2.0.0-rc6" }
-sp-core = {  path = "../../../primitives/core", version = "2.0.0-rc6" }
-sp-transaction-pool = { path = "../../../primitives/transaction-pool", version = "2.0.0-rc6" }
-prometheus-endpoint = { package = "substrate-prometheus-endpoint", path = "../../../utils/prometheus", version = "0.8.0-rc6" }
+sc-client-api = { path = "../../api", version = "2.0.0-rc5" }
+sc-consensus-babe = { path = "../../consensus/babe", version = "0.8.0-rc5" }
+sc-consensus-epochs = { path = "../../consensus/epochs", version = "0.8.0-rc5" }
+sp-consensus-babe = { path = "../../../primitives/consensus/babe", version = "0.8.0-rc5" }
+sc-keystore = { path = "../../keystore", version = "2.0.0-rc5" }
+
+sc-transaction-pool = { path = "../../transaction-pool", version = "2.0.0-rc5" }
+sp-blockchain = { path = "../../../primitives/blockchain", version = "2.0.0-rc5" }
+sp-consensus = { package = "sp-consensus", path = "../../../primitives/consensus/common", version = "0.8.0-rc5" }
+sp-inherents = { path = "../../../primitives/inherents", version = "2.0.0-rc5" }
+sp-runtime = {  path = "../../../primitives/runtime", version = "2.0.0-rc5" }
+sp-core = {  path = "../../../primitives/core", version = "2.0.0-rc5" }
+sp-api = {  path = "../../../primitives/api", version = "2.0.0-rc5" }
+sp-transaction-pool = { path = "../../../primitives/transaction-pool", version = "2.0.0-rc5" }
+sp-timestamp = { path = "../../../primitives/timestamp", version = "2.0.0-rc6" }
+tokio = { version = "0.2", features = ["rt-core", "macros", "time"] }
```

Comment:
Can you double check that we really need all of these deps? Where is `tokio` used for instance?